crates/gpui/src/platform/windows/window.rs

```diff
@@ -751,11 +754,10 @@ impl WindowsWindowInner {
             return unsafe { DefWindowProcW(self.hwnd, msg, wparam, lparam) };
         }
 
-        let dpi = unsafe { GetDpiForWindow(self.hwnd) };
-
-        let frame_x = unsafe { GetSystemMetricsForDpi(SM_CXFRAME, dpi) };
-        let frame_y = unsafe { GetSystemMetricsForDpi(SM_CYFRAME, dpi) };
-        let padding = unsafe { GetSystemMetricsForDpi(SM_CXPADDEDBORDER, dpi) };
+        // let dpi = unsafe { GetDpiForWindow(self.hwnd) };
+        let frame_x = unsafe { GetSystemMetricsForDpi(SM_CXFRAME, USER_DEFAULT_SCREEN_DPI) };
```

Comment:
I don't think we need to go to using `USER_DEFAULT_SCREEN_API` here and instead use these values in the platform titlebar which are existing TODOs

Reply:
done here. https://github.com/zed-industries/zed/pull/9449
@JunkuiZhang give it a try!